projects/1.21-fabric/assets/archers-expansion/archers_expansion/lang/zh_cn.json

```
@@ -0,0 +1,97 @@
+    "effect.archers_expansion.choking_gas": "窒息毒雾",
+    "effect.archers_expansion.choking_gas.description": "通过中毒对目标造成伤害并使其沉默。处于流血状态时伤害提升。",
+    "effect.archers_expansion.leaping_shot": "后跃射击",
+    "effect.archers_expansion.leaping_shot.description": "施法者向后跳跃，潜行时可取消。",
+    "effect.archers_expansion.disabling_shot": "致残射击",
+    "effect.archers_expansion.disabling_shot.description": "减速目标并对其持续造成伤害",
+  
+    "item.archers_expansion.tundra_hunter_spell_book": "寒霜狩典",
+    "item.archers_expansion.tundra_hunter.spell_scroll": "寒霜猎人卷轴",
+    "item.archers_expansion.tundra_hunter_chest": "寒霜外套",
+    "item.archers_expansion.tundra_hunter_head": "寒霜兜帽",
+    "item.archers_expansion.tundra_hunter_legs": "寒霜护腿",
+    "item.archers_expansion.tundra_hunter_feet": "寒霜靴子",
+    "item.archers_expansion.netherite_tundra_hunter_chest": "下界合金寒霜外套",
+    "item.archers_expansion.netherite_tundra_hunter_head": "下界合金寒霜兜帽",
+    "item.archers_expansion.netherite_tundra_hunter_legs": "下界合金寒霜护腿",
+    "item.archers_expansion.netherite_tundra_hunter_feet": "下界合金寒霜靴子",
+  
+    "spell.archers_expansion.arctic_volley.name": "寒冰箭雨",
+    "spell.archers_expansion.arctic_volley.description": "一次性射出8支寒冰箭，造成 {damage} 点伤害并冻结目标。",
+    "spell.archers_expansion.enchanted_crystal_arrow.name": "附魔水晶箭",
+    "spell.archers_expansion.enchanted_crystal_arrow.description": "造成 {damage} 点伤害，使目标眩晕并冻结。",
+    "spell.archers_expansion.frozen_pact.name": "冰霜契约",
+    "spell.archers_expansion.frozen_pact.description": "根据目标的冰霜层数造成伤害，并在 {effect_duration} 秒内降低其攻击力。",
+    "spell.archers_expansion.frozen_shot.name": "冰霜射击",
+    "spell.archers_expansion.frozen_shot.description": "你的接下来几次射击会为目标叠加冰霜效果，使其减速并逐渐冻结。",
+  
+    "effect.archers_expansion.enchanted_crystal_arrow": "水晶箭矢",
+    "effect.archers_expansion.enchanted_crystal_arrow.description": "定身目标并为其叠加冰霜层数。",
+    "effect.archers_expansion.frozen_pact": "冰霜契约",
+    "effect.archers_expansion.frozen_pact.description": "根据目标的冰霜层数造成伤害，并降低其攻击力。",
+    "effect.archers_expansion.frozen_shot": "冰霜射击",
+    "effect.archers_expansion.frozen_shot.description": "你的接下来几次射击会为目标叠加冰霜效果。",
+  
+    "item.archers_expansion.war_archer_spell_book": "战弓狩典",
+    "item.archers_expansion.war_archer.spell_scroll": "战弓卷轴",
+    "item.archers_expansion.war_archer_chest": "弓手胸甲",
+    "item.archers_expansion.war_archer_head": "弓手头盔",
+    "item.archers_expansion.war_archer_legs": "弓手护腿",
+    "item.archers_expansion.war_archer_feet": "弓手靴子",
+    "item.archers_expansion.netherite_war_archer_chest": "下界合金弓手胸甲",
+    "item.archers_expansion.netherite_war_archer_head": "下界合金弓手头盔",
+    "item.archers_expansion.netherite_war_archer_legs": "下界合金弓手护腿",
+    "item.archers_expansion.netherite_war_archer_feet": "下界合金弓手靴子",
+  
+    "spell.archers_expansion.dual_shot.name": "双重射击",
+    "spell.archers_expansion.dual_shot.description": "一次性射出两支箭",
+    "spell.archers_expansion.pin_down.name": "钉刺压制",

```

请问这里是有双关吗？为什么用了钉刺，是会有相应的粒子效果吗？直接叫压制即可。
```suggestion
    "spell.archers_expansion.pin_down.name": "压制",

```